services/offline/src/__tests__/integration.test.tsx

```diff
@@ -70,10 +71,15 @@ const TestSingleSection = (props?: any) => {
     // Props are spread so they can be overwritten
     return (
         <AlertsProvider>
-            <OfflineProvider offlineInterface={mockOfflineInterface} {...props}>
-                <TestControls id={'1'} {...props} />
-                <TestSection id={'1'} {...props} />
-            </OfflineProvider>
+            <CustomDataProvider data={{}}>
+                <OfflineProvider
+                    offlineInterface={mockOfflineInterface}
+                    {...props}
+                >
+                    <TestControls id={'1'} {...props} />
+                    <TestSection id={'1'} {...props} />
+                </OfflineProvider>
+            </CustomDataProvider>
```

Comment:
The OfflineProvider now uses a data provider to use the engine to make a data query to `/api/system/ping`

The same changes will be repeated in the next few test files